packages/daimo-userop/dist/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
todo here: these dist folders are the same as https://github.com/daimo-eth/daimo/pull/24#discussion_r1225762167, just need to wrap them into some build pipeline